README.md

```diff
@@ -1,16 +1,14 @@
-# Jenkins MATLAB Plugin
-
-The Jenkins&trade; plugin for MATLAB&reg; enables you to run your MATLAB and Simulink&reg; tests and generate artifacts such as PDF test reports, JUnit and TAP test results, and Cobertura code or model coverage reports. You also can use the plugin to export your Simulink Test&#8482; Manager results. 
+# Continuous Integration with MATLAB on Jenkins
+This plugin enables you to run MATLAB&reg; scripts, functions, and statements as part of your build. You also can run your MATLAB and Simulink&reg; tests, and generate various test and coverage artifacts such as JUnit-style test results and Cobertura code and model coverage reports.
 
+## Releases
+For a detailed list of releases, see [Change Logs](/CHANGELOG.md). 
```

Comment:
On GitHub I tend to see "Changelogs" (single word). It might just be like "setup" where the words over time have become one 🤔 
Example: https://github.blog/changelog/

Reply:
Interesting. I agree. 
@nikhilbhoski, are you fine with using "Changelogs"?